x-pack/legacy/plugins/siem/server/lib/compose/kibana.ts

```diff
@@ -37,10 +36,10 @@ import { Alerts, ElasticsearchAlertsAdapter } from '../alerts';
 
 export function compose(
   core: CoreSetup,
-  plugins: PluginsSetup,
-  env: PluginInitializerContext['env']
+  plugins: SetupPlugins,
+  isProductionMode: boolean
```

Comment:
👍 That is where we get a flag if we are running production or development mode I take it? If so, nice to know.